wuzz.go

```diff
@@ -166,6 +207,7 @@ func (a *App) Layout(g *gocui.Gui) error {
 		setViewDefaults(v)
 		v.Editable = true
 		v.Title = "URL params (F3)"
+		v.Editor = &singlelineEditor{defaultEditor}
```

Comment:
It is allowed to break lines in the GET parameter view too, so I'd suggest to use just the default editor here